docs/src/tutorial/array.rst

```diff
@@ -25,7 +25,7 @@ while the cimport adds functions accessible from Cython.
 
 A Python array is constructed with a type signature and sequence of
 initial values. For the possible type signatures, refer to the Python
-documentation for the `array module <http://docs.python.org/library/array.html>`_.
+documentation for the `array module <https://docs.python.org/3/library/array.html>`_.
```

Comment:
Same as before, switch to https but avoid requesting a specific Python version series.